auth_keycloak/examples/get_token.py

```diff
@@ -2,49 +2,80 @@
 # pylint: disable=W7935, W7936, W0403
 
 import click
-from urlparse import urljoin
+import os
+try:
+    from urlparse import urljoin
+except ImportError:
+    from urllib.parse import urljoin
+
 import json
 
 from common import (
     UID, PWD,
     CLIENT_ID, CLIENT_SECRET,
-    DATA_FILE, DOMAIN, REALM,
+    TOKEN_DATA_FILE, DEFAULTS_DATA_FILE,
+    DOMAIN, REALM,
     GET_TOKEN_PATH,
     do_request
 )
 
+default_conf = {
+    'domain': DOMAIN,
+    'realm': REALM,
+    'username': UID,
+    'password': PWD,
+    'client_id': CLIENT_ID,
+    'client_secret': CLIENT_SECRET,
+}
+
+if os.path.isfile(DEFAULTS_DATA_FILE):
+    click.echo('Loading defaults from %s' % DEFAULTS_DATA_FILE)
+    with open(DEFAULTS_DATA_FILE, 'r') as ff:
+        default_conf = json.loads(ff.read())
```

Comment:
It would be nice to overwrite only the values that exist in the json file.

Reply:
Or check if all the keys exist in the json file.

Reply:
is this comment a show stopper or should i merge?

Reply:
It's not a show stopper, you can merge.